starboard/android/shared/drm_system.cc

```diff
@@ -213,34 +212,15 @@ DrmSystem::DrmSystem(
   }
   j_media_crypto_ = env->ConvertLocalRefToGlobalRef(j_media_crypto_);
 
-  Start();
-}
-
-void DrmSystem::Run() {
-  JniEnvExt* env = JniEnvExt::Get();
-  bool result = env->CallBooleanMethodOrAbort(
-      j_media_drm_bridge_, "createMediaCryptoSession", "()Z");
-  if (result) {
-    created_media_crypto_session_.store(true);
-  }
-  if (!result && j_media_crypto_) {
-    env->DeleteGlobalRef(j_media_crypto_);
-    j_media_crypto_ = NULL;
-    return;
-  }
-
-  ScopedLock scoped_lock(mutex_);
-  if (!deferred_session_update_requests_.empty()) {
```

Comment:
These requests won't be sent if the above calls fail.  Do we have the same check in the new implementation?